### DISCUSSION TITLE
Cloning repository with `:` on Windows

### DISCUSSION THREAD

**Discussion by Pfed-prog** [git-for-windows/git#4878] [Q&A] [answered]
Gm, I am facing difficulties cloning repo that contains `:` in filenames

Here is the repository and issue that I have opened as well
https://github.com/dappradar/tokens/issues/29

[Git: files with colons in them cannot be cloned](https://jira.atlassian.com/browse/SRCTREEWIN-1574)

**Comment by dscho** [comment, marked as answer]
This is not a limitation of Git for Windows, it is a limitation of Windows itself: https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions
